TACC/Core-CMS
# feat: cmd-57 allow isolated news feed customizing

## Overview

Allow a client CMS to customize **only** the news feed **list**.

<details>

Rather than clone all of `post_list.html`, a cleint can clone and edit just the new `post_loop` block.

</details>

## Status

- [x] add block (untested)
- [x] test on tup-ui
- [x] create tup-ui PR

## Related

- [CMD-57](https://jira.tacc.utexas.edu/browse/CMD-57)
- replaces #751
- required by https://github.com/TACC/tup-ui/pull/373

## Changes

- **added** `post_loop` block to `post_list.html` template
- **moved** `post_list.html` template to `/tacc` so a client (that is not a Python app) can extend it

## Testing

1. Have custom CMS with Blog/News installed.
2. Have sample blog posts.
3. Add `taccsite_custom/tup_cms/templates/djangocms_blog/post_list.html` with
    ```
    {% extends "djangocms_blog/post_list.html" %}

    {% block post_list %}
        {% for post in post_list %}
            <h2>{{ post.title }}</h2>
        {% endfor %}
    {% endblock %}
    ```
4. Load CMS Blog/News page.
5. Verify news page shows `<h2>` title for every post, but no other content/data.

## UI

Skipped. _But would be similar to https://github.com/TACC/Core-CMS/pull/751._
